client/src/components/Button/Button.css

```diff
@@ -0,0 +1,57 @@
+.button {
+  font-style: normal;
+  font-weight: bold;
+  line-height: 20px;
+  font-family: Sahitya;
+  font-size: 15px;
+  border: 1px;
+}
+
+.button-large {
+  width: 155px;
+  height: 33px;
+  font-size: 13px;
+  line-height: 16px;
+  color: black;
+  font-family: Raleway;
+}
+
+.button-light-pink {
+  background-color: #fae5dc;
+  color: #441f0e;
+  font-weight: bold;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  height: 31px;
+  width: 88px;
```

Comment:
Can you set padding here rather than a hard height and width?

Reply:
This is for all of the buttons where you have an absolute value for width and height